### PR TITLE
dont interate when we have no entities

### PR DESCRIPTION
**Pull request by vix633**
This should address the console error we are getting, hopefully also the empty submission issue.

### CODE REVIEW

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 1]
```diff
@@ -344,9 +344,11 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
 		this._fullListLoading = true;
 
 		try {
-			var result = await this._parseActivities(entity);
-			this._data = result;
-			this._clearAlerts();
+			if (entity.entities) {
+				var result = await this._parseActivities(entity);
+				this._data = result;
+				this._clearAlerts();
```
Comment (svanherk) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Should we still be clearing alerts if the entity was changed but there just happened to be none in it? @mtjandra?

Comment (mtjandra) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Yeah it should. We should be clearing the alerts anytime we successfully load data.

Both Responses:
```
{
	"links":[],
	"entities":[]	
}
```

and 

```
{
	"links":[]
}
```


Should be supported by our app

Comment (vix633) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Pushed this.

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 2]
```diff
@@ -344,9 +344,12 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
 		this._fullListLoading = true;
 
 		try {
-			var result = await this._parseActivities(entity);
-			this._data = result;
+			if (entity.entities) {
+				var result = await this._parseActivities(entity);
+				this._data = result;
```
Comment (mtjandra) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: I think if `entity` is truthy, and `entity.entities` is falsy, you will want to set `this._data = []` to indicate there are no rows to show.

**Pull request merged**